cipd_packages/ruby/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fetches corresponding dart sdk from CIPD for different platforms, builds
+# an executable binary of codesign to `build` folder.
+#
+# This build script will be triggered on Mac code signing machines.
+
+set -e
+set -x
+
+# Get versions from config files.
+RUBY_MAJOR_VERSION=$(cat ruby_version.txt | cut -d ',' -f 1)
+RUBY_FILE_NAME=$(cat ruby_version.txt | cut -d ',' -f 2)
+COCOAPODS_VERSION=$(cat cocoapods_version.txt)
+
+# Cleanup ruby_ship/bin directory if it exists.
+rm -rf ruby_ship/bin
```

Comment:
Would you then need to create the ruby_ship dir here before you pushd?

Reply:
Actually I'll need to refactor all the scripts as the cipd tooling is expecting a `build` directory.

Reply:
Refactoring is done. Please take another look.